pkg/KubeArmorOperator/common/defaults.go

```diff
@@ -108,6 +112,7 @@ var ConfigMapData = map[string]string{
 	ConfigDefaultNetworkPosture:      "audit",
 	ConfigVisibility:                 "process,network,capabilities",
 	ConfigDefaultPostureLogs:         "true",
+	ConfigDefaultSeccompEnabled:      "true",
```

Comment:
in case if only seccomp configuration is changed in the CR, it will not be updated to the configmap, as it's not being checked in `UpdateConfigMapData()` either add it there or keept it seperate from configmap data if it's not being dynamically supported by daemonset itself.